tasks/phylogenetic_inference/augur/task_augur_mutation_context.wdl

```diff
@@ -0,0 +1,37 @@
+version 1.0
+
+task mutation_context {
+  input {
+    File refined_tree
+    File ancestral_nt_muts_json
+    String build_name
+    
+    Int cpu = 4
+    Int memory = 64
+    Int disk_size = 100
+    String docker = "us-docker.pkg.dev/general-theiagen/theiagen/nextstrain-mpox-mutation-context:2024-06-27"
+  }
+  command <<<
+    # capture version information
+    
+
+    # run augur align
+    python3 /scripts/mutation_context.py \
+      --tree ~{refined_tree} \
+      --mutations ~{ancestral_nt_muts_json} \
+      --output "~{build_name}_mpox_mutation_context.json"
+  >>>
+  output {
+    File mutation_context_json = "~{build_name}_mpox_mutation_context.json"
+  }
+  runtime {
+    docker: docker
+    memory: memory + " GB"
+    cpu: cpu
+    disks: "local-disk " + disk_size + " LOCAL"
+    disk: disk_size + " GB" # TES
+    preemptible: 0
```

Comment:
please set preemptible to `1`, as this task runs quickly and it would be a little bit cheaper to use a preemptible VM

Reply:
yep, can confirm it only took 3 min total on your test workflow and ~1 second to run the script. Preemptible is ideal for a task like this:

![image](https://github.com/theiagen/public_health_bioinformatics/assets/8172086/042df987-003e-4d0b-981e-e7a7c5a84506)